packages/react-table/src/components/Table/DraggableCell.tsx

```diff
@@ -19,7 +21,7 @@ export const DraggableCell: React.FunctionComponent<DraggableCellProps> = ({
   <Button
     id={id}
     variant="plain"
-    className={className}
+    className={css(styles.tableDraggable, className)}
```

Comment:
oh sorry, this needs to go on the parent `<td>` for this button.

Reply:
Ah I missed that detail. Updated!